src/python_tools/BUILD

```diff
@@ -2,6 +2,21 @@ package(default_visibility = ["//visibility:public"])
 
 load("@python_tools_deps//:requirements.bzl", "requirement")
 
+NOTEBOOK_DEPS = [
+    requirement("jupyter"),
+    requirement("notebook"),
+    requirement("numpy"),
+    requirement("matplotlib"),
+    requirement("bokeh"),
+    requirement("ipywidgets"),
```

Comment:
I'm wondering how this compares to using a requirements.txt file

Reply:
we actually have to do both `requirements.txt` and this for PyPI dependencies under Bazel